src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,34 @@
+.stars {
+  display: flex;
+  height: 16px;
+  justify-content: space-between;
+  margin-bottom: 24px;
+
+  &--4 {
+    gap: 4px;
```

Comment:
The `gap` property should be defined within the `.stars` block, not only for the `.stars--4` modifier. This is because all star blocks likely need a gap, and we should keep the styles consistent.